narwhals/_dask/expr.py

```diff
@@ -244,6 +244,12 @@ def sum(self) -> Self:
             "sum",
         )
 
+    def round(self) -> Self:
+        return self._from_call(
+            lambda _input: _input.round(),
+            "round",
+        )
```

Comment:
ah you're missing a `decimals` argument

Reply:
@MarcoGorelli  I see that now. I would have submitted a resolution sooner but my government basically shutdown the internet for a view days.

Reply:
> ah you're missing a `decimals` argument

I think  i have resolved it in my new push